src/send/SendAmount/SendAmountHeader.tsx

```diff
@@ -71,7 +72,12 @@ function SendAmountHeader({
   return (
     <>
       <CustomHeader
-        left={<BackButton eventName={backButtonEventName} />}
+        left={
+          <BackButton
+            eventName={backButtonEventName}
+            style={{ paddingLeft: variables.contentPadding + 6 }}
```

Comment:
Can we make `CustomHeader` include the required padding instead?

Reply:
I agree that we should make this change, but it seems outside of the scope of the PR. I'd rather make the changes in their own PR to avoid more changes in this PR. I have a stash with those changes and would be happy open a PR to standardize the headers.